src/main/scala/mg7/loquats/5.merge.scala

```diff
@@ -1,37 +1,33 @@
 package ohnosequences.mg7.loquats
 
 import ohnosequences.mg7._
-
 import ohnosequences.loquat._
-
 import ohnosequences.statika._
-
 import ohnosequences.cosas._, types._, klists._
-
 import ohnosequences.datasets._
-
 import better.files._
 
-
 case object mergeDataProcessing extends DataProcessingBundle()(
-  input = data.mergeInput,
-  output = data.mergeOutput
-) {
+  input   = data.mergeInput,
+  output  = data.mergeOutput
+)
+{
 
   def instructions: AnyInstructions = say("Merging, joining, amalgamating!")
 
-
   // TODO: use streams, file-writers, etc. stuff
+  // TODO no default arguments please
   def mergeChunks(dir: File, out: File, header: Option[String] = None): Unit = {
+    // TODO foreach on option :|
```

Comment:
First Option here does not add anything; I don't think you'd like an empty line for `Some("")`.